src/streaming/Stream.js

```diff
@@ -714,6 +714,8 @@ function Stream(config) {
     }
 
     function getProcessors() {
```

Comment:
getProcessors is used in other functions as well. I would prefer to add a flag to this function "includeTextIfDisabled" or something similar. This flag should default to true and should only be false for the use case you describe.